src/modules/map/components/footer/MapFooterAttribution.vue

```diff
@@ -68,35 +72,54 @@ export default {
     },
     watch: {
         lang(lang) {
-            this.externalDisclaimerTooltip?.forEach((instance) => {
-                instance.setContent(this.getExternalDisclaimerPopupContent())
-            })
+            this.updateDataDisclaimer()
         },
     },
     updated() {
         // We need to destroy and recreate the tippy tooltip on each update
         // otherwise when removing/adding the external layer (e.g. visibility toggle) the
         // tippy won't work anymore.
-        this.externalDisclaimerTooltip?.forEach((instance) => {
-            instance.destroy()
-        })
-        this.externalDisclaimerTooltip = tippy('.external-source', {
-            content: this.getExternalDisclaimerPopupContent(),
-            arrow: true,
-            placement: 'top',
-        })
+        this.updateDataDisclaimer()
     },
     unmount() {
-        this.externalDisclaimerTooltip?.forEach((instance) => {
+        this.dataDisclaimerTooltip?.forEach((instance) => {
             instance.destroy()
         })
     },
     methods: {
         getAttributionWithComaIfNeeded(attribution, index) {
             return `${attribution}${index !== this.sources.length - 1 ? ',' : ''}`
         },
-        getExternalDisclaimerPopupContent() {
-            return this.$i18n.t('external_data_tooltip')
+        getDataDisclaimerPopupContent(sourceName) {
+            return this.$i18n.t('external_data_warning').replace('--URL--', sourceName)
+        },
+        updateDataDisclaimer() {
+            this.dataDisclaimerTooltip?.forEach((instance) => {
+                instance.destroy()
+            })
+            this.dataDisclaimerTooltip = []
+            this.sources.forEach((source) => {
+                this.dataDisclaimerTooltip.concat(
+                    tippy(`#source-${source.id}.data-disclaimer`, {
+                        theme: 'primary',
+                        content: this.getDataDisclaimerPopupContent(source.name),
+                        arrow: true,
+                        placement: 'top',
+                        hideOnClick: false,
+                        trigger: 'mouseenter focus click',
+                        onShow: (instance) => {
+                            setTimeout(() => {
+                                instance.hide()
+                            }, 5000)
```

Comment:
we could use the getter `isDesktopMode` to only hide after 5sec when we show the mobile menu, what do you think?

Reply:
This has been solved in #362